docs/tutorials/code/chisel/rockcraft.yaml

```
@@ -1,17 +1,16 @@
-name: chisel-openssl
-summary: OpenSSL from Chisel slices
-description: A "bare" rock containing an OpenSSL installation created from Chisel slices.
+name: chisel-hello
+summary: Hello world from Chisel slices
+description: A "bare" rock containing hello binaries from Chisel slices.
 license: Apache-2.0
 
-version: "0.0.1"
+version: "1.0"
```

```suggestion
version: "latest"
```

better use latest so we don't need to worry about which version of `hello` is being installed''

---

same for the 1st tutorial